src/netspresso_trainer/models/full/experimental/pidnet.py

```diff
@@ -38,10 +38,10 @@ def __init__(
         self.conv1 = nn.Sequential(
             ConvLayer(in_channels=3, out_channels=planes,
                       kernel_size=3, stride=2, padding=1,
-                      norm_type='batch_norm', act_type='relu'),
+                      norm_type='batch_norm', act_type='relu', bias=True),
```

Comment:
The custom convolution layer has 'bias=False' set as the default value. 
However, in '[torch.nn.Conv2d](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html)', 'bias=True' is the default value. 

Could you please clarify the reason behind setting 'bias=False' as the default value in the custom layer?

Reply:
Our custom `ConvLayer` has conv-bn-act pattern.
And, convolution before batch normalization usually has no bias because batch normalization takes a role same with convolution bias, thus it cancels effect of the bias.
Please refer [Batch normalization paper](https://arxiv.org/pdf/1502.03167.pdf) section 3.2 second paragraph.

Reply:
Thank you for the clarification.